models/storyboard.js

```diff
@@ -19,14 +19,22 @@ function createNav(items) {
     const nav = [];
 
     items.forEach((item, id) => {
-        const [mediaType] = Object.keys(item);
-        // fallback for now until every type has a title field to fill in. (or something for nav)
-        const tmpTitle = item[mediaType].title ? item[mediaType].title : `${id} ${mediaType}`;
-        const title = tmpTitle.replace(/<\/?[^>]+(>|$)/g, "");
+      var elKey = Object.keys(item)[0];
+      const root = item[elKey];
+      var writeToNav = true;
+      if (root && root.in_nav && root.in_nav.suppress) {
+        writeToNav = false;
+      }
+      const [mediaType] = Object.keys(item);
+      // fallback for now until every type has a title field to fill in. (or something for nav)
+      const tmpTitle = item[mediaType].title ? item[mediaType].title : `${id} ${mediaType}`;
+      const title = tmpTitle.replace(/<\/?[^>]+(>|$)/g, "");
```

Comment:
this might leave garbage in the title, if it's html better use innerText

Reply:
I've tried this but it's actually just a text string objects with html like entities in. This also strips all MS:O guff as well. I'd be happy for someone to take it over and sanitise if they wanted.